tests/macros/InterfaceOverview.test.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Test the API Interface Overview page content macro
+ *
+ * @prettier
+ */
+
+// Get necessary modules
+const {
+    assert,
+    describeMacro,
+    beforeEachMacro,
+    itMacro,
+    readJSONFixture
+} = require('./utils');
+const { JSDOM } = require('jsdom');
+
+const INTERFACE_DATA = JSON.stringify([
+    {
+        XMLHttpRequestEventTarget: {
+            inh: 'EventTarget',
+            impl: []
+        }
+    }
+]);
+
+const DATA = ((...dataNames) => {
+    /** @type {Record<string, any>} */
+    let result = {};
+    for (const name of dataNames) {
+        result[name] = readJSONFixture('interface-overview', name);
+    }
+    return result;
+})('EventTarget', 'XMLHttpRequestEventTarget');
+
+/**
+ * @param {DocumentFragment} dom
+ * @param {string[]} headers
+ */
+function checkSectionTitles(dom, headers) {
+    // An unclosed <dl> would just wrap all the remaining <h2>s,
+    // which would no longer be directly accessible through
+    // the `dom.children` property, as they’re no longer
+    // direct descendants.
```

Comment:
I know I asked a comment about whether these tests were actually testing the dl bug fix. So thank you for explaining this to me. But this comment here won't make any sense to any future reader who has not read this conversation on github, so please remove these lines.  

Instead, you could add something to the doc comment that just says that it verifies that dom element has h2 elements as direct children with content (and in the same order) as the headers array.